org.eclipse.xtext.builder.tests/src/org/eclipse/xtext/builder/tests/ParallelBuildEnabledTest.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2018 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.builder.tests;
+
+import static org.junit.Assert.*;
+
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.preferences.DefaultScope;
+import org.eclipse.xtext.builder.internal.Activator;
+import org.eclipse.xtext.builder.preferences.BuilderPreferenceAccess;
+import org.junit.Test;
+
+/**
+ * @author Christian Dietrich - Initial contribution and API
+ */
+public class ParallelBuildEnabledTest {
+
+	@Test
+	public void testIsParallelBuildEnabledInWorkspace() {
+		int maxConcurrentProjectBuilds = DefaultScope.INSTANCE.getNode(ResourcesPlugin.PI_RESOURCES).getInt("maxConcurrentBuilds", -1);
```

Comment:
Should be instance scope if I think over it

Reply:
still holds true

Reply:
sorry, missed that. updated.